vinyl/display.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+//link database to page 
+$db = new PDO('mysql:host=db; dbname=Vinyl', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I'd suggest making this into a function! 

The function would be in charge of connecting to your database, and then returning $db. This will make life easier in the future when it comes to connecting to the database on different pages in your application.